lib/home/surveys/take_survey/survey_theme.dart

```diff
@@ -99,19 +100,25 @@ class SurveyStyling extends StatelessWidget {
         ),
       ),
       child: Scaffold(
-        body: SingleChildScrollView(
-          child: Container(
-            decoration: BoxDecoration(
-              gradient: LinearGradient(
-                begin: const Alignment(-0.25, -1.0),
-                end: const Alignment(0.25, 1.0),
-                colors: [colors.surface, colors.background],
+        body: Builder(
+          builder: (context) {
+            final colors = context.colorScheme;
+            final size = MediaQuery.sizeOf(context);
```

Comment:
Using a `Builder` here lets us benefit from the animated `ColorScheme`.

Normally, I'd prefer to use a `LayoutBuilder` rather than the builder with a `MediaQuery` to get the screen size, since it takes the [`SafeArea`](https://api.flutter.dev/flutter/widgets/SafeArea-class.html) into account. But in this case, we want the gradient to overlap the safe area, so it works.